packages/dd-trace/test/plugin_manager.spec.js

```diff
@@ -240,6 +240,17 @@ describe('Plugin Manager', () => {
       loadChannel.publish({ name: 'four' })
       expect(instantiated).to.deep.equal(['two', 'four'])
     })
+    it('configures service naming schema resolution', () => {
```

Comment:
I wonder if we should break this up:


```js
const Nomenclature = require('../../dd-trace/src/service-naming') // top of file?

describe('configures service naming schema resolution', () => {
      const config = {
        foo: { 'bar': 1 },
        baz: 2
      }
      let configureSpy
  beforeEach(() => {
   configureSpy = sinon.spy(Nomenclature, 'configure')
  })
  afterEach(() => {
      configureSpy.restore()
  })
    it('works', () => {
      pm.configure(config)
      expect(configureSpy).to.have.been.calledWith(config)
    })
})
```

The `expect()` call throws IIRC, and basically the test can fail for other reasons and not get to the part with the restore. By throwing the cleanup in an `afterEach` (or `after`) it then prevents a situation where a failed expectation pollutes the future state of the tests.

Reply:
Done in #2941